plugins/tst/src/main/java/kg/apc/jmeter/timers/VariableThroughputTimer.java

```diff
@@ -35,7 +36,7 @@
     public static final int DURATION_FIELD_NO = 2;
     public static final int FROM_FIELD_NO = 0;
     public static final int TO_FIELD_NO = 1;
-    private static final Logger log = LoggingManager.getLoggerForClass();
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(VariableThroughputTimer.class);
```

Comment:
Changing logger is not something planned for this plugin, since it makes it unusable on old versions of JMeter.

Reply:
Hi @undera ,
Are you sure reverting SLF4J is needed ?
As slf4j is a transitive dependency it will work in 2.13.
Also, 3.0 and 2.13 represent 10% and IMO users should be encouraged to move to last JMeter or at  least not be before 3.1.

But if you want me to revert I'll do.

Regards